### DISCUSSION TITLE
Team dumping tokens

### DISCUSSION THREAD

**Discussion by Neekel** [arcana-network/discussions#23] [General]
Tomorrow 04.04.24 $XAR token was listed on Bybit and Gate. 
Immediately after listing, the team brazenly started dumping their entire volume of tokens and completely unloaded into their community, spitting on them. 
After the CEO's fiery promise to lock in the team's tokens for a year, they continued to dump their tokens from the smart contract to the exchanges and exited the chat rooms.
People who bought token at $1 had to sell them at $0.2. Greedy members of the team took their own commuity's money besides investors' money.
That's the story.

**Comment by anivenu** [comment]
Hi @Neekel 

Understand the concern but to be clear the only tokens that have been unlocked during TGE are for investors and the ecosystem. You can confirm this by looking on-chain yourself. The team's tokens have not yet been unlocked and our team is still very much active on all of our channels of communication (telegram, discord). 

We will be introducing staking in the coming week where you can earn more rewards for being an early Arcana adopter and have many more exciting things planned for the coming weeks & months.
